### PR TITLE
feat: badge component

### PR DESCRIPTION
**Pull request by harshitadaswani**
added profile badges and number badges in the badge component

### CODE REVIEW

File: components/badge.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+.online {
+    background-color: green;
+    bottom: 0.3rem;
+    left: 3rem;
+}
+
+.offline {
+    background-color: grey;
```
Comment (rkapoor10) [inline components/badge.css]: use css variables for colors

File: components/badge.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    bottom: 0.3rem;
+    left: 3rem;
+}
+
+.dnd {
+    background-color: red;
```
Comment (rkapoor10) [inline components/badge.css]: same as above

File: components/badge.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+                    <li class="p-xs m-xs"><a href="button.html">Button</a></li>
+                    <li class="p-xs m-xs"><a href="card.html">Card</a></li>
+                    <li class="p-xs m-xs"><a href="image.html">Image</a></li>
+                    <li class="p-xs m-xs"><a href="input.html">Input</a></li>                    
+                </ul>
+
+            </aside>
+
+        <main class="main-container">
+            <h1 class="h1-title">Badge</h1>
```
Comment (rkapoor10) [inline components/badge.html:51]: if h1-title gives colors name it accordingly

File: components/badge.html [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+            <p class="border-bottom">Badges are small circles, positioned either at top-right or bottom-right of the parent component. Badge can be used to display numbers, online / offline status, depending on where they are used.</p>
+            <h2 class="h2-title">Profile Badge</h2>
```
Comment (rkapoor10) [inline components/badge.html:53]: same as above

File: components/badge.html [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+                    <img src="../assets/profile-pic.svg" alt="male-profile" class="avatar">
+                    <div class="dnd"></div>
+                </div>    
+                <div class="badge">
+                    <img src="../assets/male-avatar.svg" alt="male avatar" class="avatar">
+                    <div class="offline"></div>
+                </div>            
+            </div>
+            <iframe src="https://www.thiscodeworks.com/embed/61faef71b783be0015bbaf6c" style="width: 95%; height: 20rem;" class="code-snippet" frameborder="0"></iframe>
+            <div class="border-bottom"></div>
```
Comment (rkapoor10) [inline components/badge.html:70]: what is the use of this empty div?

File: components/badge.html [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+            <h2 class="h2-title">Number Badge</h2>
+            <p>Number Badge is used to show the number of items present in the parent component.</p>
+            <div class="example-div br-l">
+                <div class="badge">
+                    <i class="fas fa-envelope icon"></i>
+                    <div class="number-badge">10</div>
```
Comment (rkapoor10) [inline components/badge.html]: separate out classes for different properties...if possible.

File: stylesheets/base.css [hunk 7]
```diff
@@ -69,7 +69,7 @@ img,iframe {
 }
 
 [class^="f"] {
-    font-size: inherit;
+    font-size: 2rem;
```
Comment (rkapoor10) [inline stylesheets/base.css]: find a way around...don't use fixed font size.

**Review by rkapoor10 — CHANGES_REQUESTED** [review]
Refactor the code according to the given comments. Rest all is good.

File: components/badge.html (hunk 3 above)
Comment (harshitadaswani) [reply components/badge.html:51]: h1-title is a utility CSS class having styling related to the title elements

File: components/badge.html (hunk 4 above)
Comment (harshitadaswani) [reply components/badge.html:53]: h1-title is a utility CSS class having styling related to the title elements

File: components/avatar.css [hunk 8]
```diff
@@ -26,7 +26,6 @@
     padding: var(--l-size);
     display: flex;
     flex-flow: row wrap;
```
Comment (mariyasada) [inline components/avatar.css:28]: delete that unnecessary comment

**Review by mariyasada — APPROVED** [review]

File: components/avatar.css (hunk 8 above)
Comment (harshitadaswani) [reply components/avatar.css:28]: already deleted! thanks

**Review by sks-21 — APPROVED** [review]
Everything looks nice. Good to merge !

**Review by rkapoor10 — APPROVED** [review]
Great! good to merge.

**Pull request merged**